internal/dev_server/dev_server.go

```diff
@@ -10,8 +10,7 @@ import (
 	"github.com/launchdarkly/ldcli/internal/dev_server/db"
 )
 
-// FIXME this is gonna get replaced by a cli command
-func main() {
+func RunServer() {
```

Comment:
@mike-zorn rather than porting this over I think it maybe makes more sense to leave this here and export it into the cmd, what do you think?

Reply:
works for me!